CHANGELOG.md

```diff
@@ -4,89 +4,92 @@
+- Add multiple import at once for Android
+- Remove Pouch adapter migration
 
 ## 🐛 Bug Fixes
 
-* Do not update files in parallel in the qualification migration service, as it might fail in nsjail for too many files
-* Fix MoveModal breadcrumb
-* Display reasons of incorrect file name (illegal characters, forbidden name)
-* Prevent errors during upload of file inside Dropzone
-* Handle better icon inside the searchbar
-* Upgrade cozy-client in order to fix albums page from photos
+- Do not update files in parallel in the qualification migration service, as it might fail in nsjail for too many files
+- Fix MoveModal breadcrumb
+- Display reasons of incorrect file name (illegal characters, forbidden name)
+- Prevent errors during upload of file inside Dropzone
+- Handle better icon inside the searchbar
+- Upgrade cozy-client in order to fix albums page from photos
 
 ## 🔧 Tech
 
-* Use `<SharingBannerPlugin />` and `useSharingInfos()` from `cozy-sharing` instead of internal components
-* Fixed an error in Search result when the result contained at least one Cozy Note
-* Update cordova to 8.1.2 and cordova-android to 9.1.0
-* Upgrade cozy-client, cozy-scanner caniuse-lite and fix tests
-* Upgrade cozy-sharing to fix typo in French
-* Add locales in gitignore
-* Explicit full path when importing cozy-ui component inside doc
+- Use `<SharingBannerPlugin />` and `useSharingInfos()` from `cozy-sharing` instead of internal components
+- Fixed an error in Search result when the result contained at least one Cozy Note
+- Update cordova to 8.1.2 and cordova-android to 9.1.0
+- Upgrade cozy-client, cozy-scanner caniuse-lite and fix tests
+- Upgrade cozy-sharing to fix typo in French
+- Add locales in gitignore
+- Explicit full path when importing cozy-ui component inside doc
 
 # 1.37.0
 
 ## 🐛 Bug Fixes
 
-* Fixed an error on mobile that was preventing users to long tap in order to trigger multiple files selection
-* Fixed an error in directory tree names appearing under filenames where sometimes, the path appeared scrambled
-* Fixed an error where creating a directory sent two save actions instead of one
-* Added a missing loading status on delete confirm modal button
-* Fixed issues related to recent view not going where it should when navigating back and forth in directory paths
+- Fixed an error on mobile that was preventing users to long tap in order to trigger multiple files selection
+- Fixed an error in directory tree names appearing under filenames where sometimes, the path appeared scrambled
+- Fixed an error where creating a directory sent two save actions instead of one
+- Added a missing loading status on delete confirm modal button
+- Fixed issues related to recent view not going where it should when navigating back and forth in directory paths
 
 ## 🔧 Tech
 
-* Add CodeQL in order to scan the code 🚫
-* Add rel noopener on target blank link
+- Add CodeQL in order to scan the code 🚫
+- Add rel noopener on target blank link
```

Comment:
Oh no, why all these changes ?

Reply:
in markdown, the result is the same, isn't it?

Reply:
Wops, looks like an automatic formatting

Reply:
It does not change the result indeed, but I prefer not making such change :) [4231221](https://github.com/cozy/cozy-drive/pull/2532/commits/423122139676c714016da6c869071f4f37803bf4)